api/config/cdr_config_local.json

```diff
@@ -74,6 +82,23 @@
       "hasFitbitData": true,
       "hasCopeSurveyData": true,
       "accessTier": "registered"
+    },
+    {
+      "cdrVersionId": 5,
+      "isDefault": false,
+      "name": "CDR for Controlled Tier - DO NOT USE",
+      "dataAccessLevel": 1,
+      "bigqueryProject": "fc-aou-cdr-synth-test",
+      "bigqueryDataset": "SR2019q4r3",
+      "microarrayBigqueryDataset": "microarray_data",
+      "creationTime": "2020-08-26 00:09:51Z",
```

Comment:
how this date get generated?

Reply:
copy/paste.  this date is a lie.  :)

I can fix it if you like.

Reply:
I think it doesn't matter?